### PR TITLE
Add flexbox documentation

### PR DESCRIPTION
**Pull request by JulienVannier66**
### What does this PR do?

Add flexbox documentation

Related to (nothing)

### What are the observable changes?
Peek 2021-09-15 15-47

### Good PR checklist

- [x] Title makes sense
- [x] Is against the correct branch
- [x] Only addresses one issue
- [x] Properly assigned
- [x] Added/updated documentation
- [x] Properly labeled

### CODE REVIEW

File: less/utilities/_flex.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+### Main axis alignment
+
+`fx-malign-start` class equivalent of `{justify-content: flex-start;}`
+
+`fx-malign-center` class equivalent of `{justify-content: center;}`
+
+`fx-malign-end` class equivalent of `{justify-content: flex-end;}`
+
+`fx-malign-space-between` class equivalent of `{justify-content: space-between;}`
```
Comment (Miexil) [inline less/utilities/_flex.stories.mdx:44]: Should we give more details about what `space-between` `space-around` and `space-evenly` do or is that clear for everyone ?

File: less/utilities/_flex.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+
+### Cross axis alignment
+
+`fx-xalign-start` class equivalent of `{align-items: start;}`
+
+`fx-xalign-center` class equivalent of `{align-items: center;}`
+
+`fx-xalign-end` class equivalent of `{align-items: end;}`
+
+### Exemple
```
Comment (Miexil) [inline less/utilities/_flex.stories.mdx:58]: Example

File: less/utilities/_flex.stories.mdx [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+  }}
+/>
+
+# Flexbox
+
+The Flexbox system aims to provide a more efficient way of arranging, aligning and distributing space between the elements
+of a container. The goal of the flexible layout is to give the container the ability to change the width and height of its
+elements to best fill the available space.
+
+Keep in my mind that the Alignment axis are switched based on the flex-direction:
```
Comment (Miexil) [inline less/utilities/_flex.stories.mdx:17]: Plural of axis is `axes`

But maybe re-write the sentence as below :
`Keep in mind that the alignment of the axes are swtiched based on the flex-direction:`

**Review by Miexil — APPROVED** [review]

File: less/utilities/_flex.stories.mdx (hunk 1 above)
Comment (Miexil) [reply less/utilities/_flex.stories.mdx:44]: I guess its clear enough with the examples below !
Disregard this comment 🙂

**Review by Miexil — APPROVED** [review]

**Review by phndiaye — APPROVED** [review]

**Pull request merged**